frontend/src/components/pages/Scheduling/ConfirmDetails.tsx

```diff
@@ -295,10 +327,26 @@ const ConfirmDetails = ({
             </Text>
             <Text textStyle="mobileBody">{currentSchedule.notes}</Text>
           </Box>
+          {currentSchedule.volunteerId && volunteer && (
+            <Box>
+              <Text textStyle="mobileSmall" color="hubbard.100" pt="1.4em">
+                Assigned volunteer
+              </Text>
+              <Text textStyle="mobileBody">{`${volunteer.firstName} ${volunteer.lastName}`}</Text>
+              <Text textStyle="mobileSmall" color="hubbard.100" pt="1.4em">
+                Email
+              </Text>
+              <Text textStyle="mobileBody">{volunteer.email}</Text>
+              <Text textStyle="mobileSmall" color="hubbard.100" pt="1.4em">
+                Phone
+              </Text>
+              <Text textStyle="mobileBody">{volunteer.phoneNumber}</Text>
+            </Box>
+          )}
```

Comment:
I believe when there is no volunteer assigned, we still show this information but with a hyphen. Look at the following from the designs: 
<img width="590" alt="image" src="https://user-images.githubusercontent.com/37384882/163651604-14075787-2f45-4d4b-9486-97f733812b43.png">

From the scheduling flow, but should be the same view details screen as well.